### ISSUE TITLE
Improvements to export to google buckets dialog - Why specify dataframes for different summaries in the main dialog and then repeat the same task in the tabs within the summaries sub-dialog?

### ISSUE THREAD

**Issue by jkmusyoka** [IDEMSInternational/R-Instat#9101]
@Vitalis95 
Since users have to specify/select/provide the dataframes containing the different types of summaries in the main dialog, is it possible to link these selections to the dataframe selectors in the tabs of the subsequent summaries sub-dialog so that users do not have to repeat this task again? This would be efficient and make the dialog easier-to-use!

**Comment by rdstern** [comment]
@Vitalis95 I agree this does not need to be checked twice.  I am happy either way, i.e. either delete the need on the main dialog. Then you have to choose the dataframe on the sub-dialog.  Or continue as now, specifying on the main dialog and it is then fixed on the sub-dialog.  (I think that's what happens when you use the Data Options (on each dialog) and then go to the filter dialog?

**Comment by Vitalis95** [comment]
@lilyclements , will it affect the function if we delete the data frames  that are being checked twice in the main dialog and only have them only in the sub dialog?

**Comment by lilyclements** [comment]
The sub-dialog checks are for when you’re sending the summaries data, whereas the main dialog are for sending the definitions. Since we always send the definitions and only sometimes send the summaries, we I’d suggest we want any on the main dialog to stay on the main, but they can be removed from the sub (provided they’re still called in the correct function)
